backend/tests/test_classify_tests.py

```
@@ -0,0 +1,494 @@
+    out = capsys.readouterr().out
+    assert rc == 0
+    assert "0 tests classified" in out
+
+
+# ---------------------------------------------------------------------------
+# AC-1.3 — REPORT mode (PF-Q1 sample generation)
+# ---------------------------------------------------------------------------
+
+
+@pytest.mark.unit
+def test_report_mode_writes_sample_to_default_path(tmp_path, monkeypatch):
```

Ruff (0.15.13)

[warning] Unused function argument: `monkeypatch`

(ARG001)